diff --git a/Python/setup.py b/Python/setup.py
index 714b2bd..c63d276 100644
--- a/Python/setup.py
+++ b/Python/setup.py
@@ -21,9 +21,6 @@ COMPUTE_CAPABILITY_ARGS = [  # '-gencode=arch=compute_20,code=sm_20', #deprecate
     '-gencode=arch=compute_52,code=sm_52',
     '-gencode=arch=compute_60,code=sm_60',
     '-gencode=arch=compute_61,code=sm_61',
-    '-gencode=arch=compute_70,code=sm_70',
-    '-gencode=arch=compute_75,code=sm_75',
-    '-gencode=arch=compute_86,code=sm_86',
     '--ptxas-options=-v', '-c',
     '--default-stream=per-thread',
     ]
