diff --git a/Python/setup.py b/Python/setup.py
index 714b2bd..52fd035 100644
--- a/Python/setup.py
+++ b/Python/setup.py
@@ -22,8 +22,7 @@ COMPUTE_CAPABILITY_ARGS = [  # '-gencode=arch=compute_20,code=sm_20', #deprecate
     '-gencode=arch=compute_60,code=sm_60',
     '-gencode=arch=compute_61,code=sm_61',
     '-gencode=arch=compute_70,code=sm_70',
-    '-gencode=arch=compute_75,code=sm_75',
-    '-gencode=arch=compute_86,code=sm_86',
+    '-gencode=arch=compute_70,code=compute_70',
     '--ptxas-options=-v', '-c',
     '--default-stream=per-thread',
     ]
@@ -370,10 +369,6 @@ setup(name='pytigre',
       ext_modules=[Ax_ext, Atb_ext, tvdenoising_ext, minTV_ext, AwminTV_ext],
       py_modules=['tigre.py'],
       cmdclass={'build_ext': BuildExtension},
-      install_requires=['Cython',
-                        'matplotlib',
-                        'numpy',
-                        'scipy'],
       license_file='LICENSE.txt',
       license='BSD 3-Clause',
       # since the package has c code, the egg cannot be zipped
